### ISSUE TITLE
How to run on none task data

### ISSUE THREAD

**Issue by dsm-72** [bunnech/jkonet#2]
From `main.py` it is unclear how to run on one's own data

**Comment by bunnech** [comment]
For this you need to create your own dataloader suitable for your dataset and add it to the pipeline, see https://github.com/bunnech/jkonet/blob/main/main.py#L35. Examples of dataloaders are provided in `jkonet/data`.

Best,
Charlotte

**Comment by dsm-72** [comment]
@bunnech Yes I noticed this. However it appears that rather than importing / overriding the class, due to relative imports, one must actually alter the code under jkonet/data to specify the data and while there are some examples of toy data there (like line and circle) it is still unclear on the requirements needed for this data. e.g. how to specify time points, how many dimensions the input can be in, etc
